src/VisualStudio/Core/Impl/Options/Style/EnumCodeStyleOptionViewModel.cs

```diff
@@ -42,6 +58,21 @@ static EnumCodeStyleOptionViewModel()
             OptionSet options,
             string groupName,
             List<CodeStylePreference> preferences)
+            : this(option, null, description, enumValues, previews, info,
```

Comment:
`language: null` (I know it's annoying to have to do all the other ones too... For now...)